src/pages/_app.tsx

```diff
@@ -48,7 +48,7 @@ const USSFPortalApp = ({
     <ApolloProvider client={client}>
       <AnalyticsProvider>
         <AuthProvider>
-          <ThemeProvider enableSystem={false}>
+          <ThemeProvider enableSystem={false} attribute={'data-color-theme'}>
```

Comment:
To use the storybook plugin, the attribute needs to be `data-color-theme` instead of `data-theme`. Small change for very helpful functionality.